CMakeLists.txt

```diff
@@ -182,7 +182,7 @@ if(BUILD_GUI)
   if(BUILD_GUI_TESTS)
     list(APPEND qt_components Test)
   endif()
-  find_package(Qt5 5.11.3 MODULE REQUIRED
+  find_package(Qt 6.2 MODULE REQUIRED
```

Comment:
Would the minimum QT version in `doc/dependencies.md` need to be updated?

Reply:
Thanks! Updated.